source/views/main.js

```diff
@@ -769,6 +769,12 @@ gestureStart: function(inSender, e){
 	this.previousScale = e.scale;
 	this.previousS = 0;
 
+	e.centerX = 0;
```

Comment:
This seems to break stuff on legacy somehow:

2016-01-20T19:15:40.554948Z [17474] webos-device user.crit LunaSysMgr: {LunaSysMgrJS}: org.webosports.app.maps: Uncaught TypeError: Cannot read property 'length' of undefined, file:///media/cryptofs/apps/usr/palm/applications/org.webosports.app.maps/build/app.js:321
2016-01-20T19:15:40.708275Z [17474] webos-device user.crit LunaSysMgr: {LunaSysMgrJS}: org.webosports.app.maps: Uncaught TypeError: Cannot read property 'cx' of undefined, file:///media/cryptofs/apps/usr/palm/applications/org.webosports.app.maps/build/app.js:321

Google Location services seems broken on legacy, but this piece of code seems to break the pinch & zoom altogether on legacy now.